lib/variations.js

```diff
@@ -17,7 +20,7 @@ const variations = {
 };
 
 function isEligible() {
-  return true;
+  return !Preferences.isSet('services.sync.username'); // users with sync are ineligible
```

Comment:
A utility function `isSyncSetup` would be appropriate. Similarly, any place where you're dealing with the details of detecting sync (reading the username preference or watching it) could be abstracted away. Like:

```js
isSyncSetup();
onSyncChange((syncSetup) => {
  if (syncSetup) {
    // blah
  }
});
```

Then again, this doesn't seem likely to change, and it isn't that hard to understand. Maybe just add the `isSyncSetup` helper?